backend/dataall/modules/shares_base/services/share_logs_service.py

```diff
@@ -15,8 +16,18 @@
 
 class ShareLogsService:
     @staticmethod
-    def check_view_log_permissions(username, groups, shareUri):
-        with get_context().db_engine.scoped_session() as session:
+    @is_feature_enabled_for_allowed_values(
+        allowed_values=['admin-only', 'enabled', 'disabled'],
+        enabled_values=['admin-only', 'enabled'],
+        default_value='enabled',
+        config_property='modules.shares_base.features.show_share_logs',
+    )
+    def check_view_logs_permissions(username, groups, shareUri):
```

Comment:
nit: we do ont need to pass the username and groups. We can get them directly from the context object `context.groups` and `context.username`